app/controllers/domains_controller.rb

```diff
@@ -0,0 +1,23 @@
+class DomainsController < ApplicationController
+  before_action :require_logged_in_admin
+  before_action :set_domain, only: [:update, :edit]
+
+  def edit; end
+
+  def update
+    banned_reason = params.dig('domain', 'banned_reason')
+    if banned_reason.present?
```

Comment:
I forgot to mention in the issue, but it should also be possible to unban domains via the edit form. If you want to include that functionality I'd appreciate it, but I see how it'd require touching the model as well, and I can add that to a separate issue after merging this PR if you don't want to take it.

Reply:
Let me add the 'unban' functionality in this PR, please

Reply:
I'd appreciate it, thank you!